addons/fleet/views/fleet_vehicle_model_views.xml

```diff
@@ -219,22 +204,16 @@
                         </a>
                         <a role="menuitem" t-if="widget.deletable" type="delete" class="dropdown-item">Delete</a>
                     </t>
-                    <t t-name="kanban-box">
-                        <div class="oe_kanban_vignette oe_semantic_html_override oe_kanban_global_click">
-                            <div class="o_kanban_image">
-                                <img alt="img" t-att-src="kanban_image('fleet.vehicle.model.brand', 'image_128', record.id.raw_value)" class="o_image_64_max" height="52"/>
-                            </div>
-                            <div class="oe_kanban_details">
-                                <h4 class="oe_partner_heading">
-                                    <a type="open" class="o_kanban_record_title">
-                                        <field name="name"/>
-                                    </a>
-                                </h4>
-                                <div>
-                                    <field name="model_count"/> MODELS
-                                </div>
+                    <t t-name="kanban-card" class="flex-row">
+                        <aside class="w-25 pe-1">
+                            <field name="image_128" widget="image"/>
+                        </aside>
+                        <main class="w-100">
+                            <a type="open" class="fw-bold fs-5 text-black"><field name="name"/></a>
```

Comment:
the whole card is clickable, so this link is not necessary

Reply:
this link will open the form view of car name and logo(fleet_vehicle_model_brand_view_form) and whole card will open the list view(fleet_vehicle_model_view_tree).

Reply:
Ok I didn't notice that there was an action set on the kanban root node. I checked it with cth, and it is actually a mistake. You should revert the diff (in this file) of commit https://github.com/odoo/odoo/commit/8313a66a522cce3ceb81ca8ede07255785c4431f. There's a fix to do though: the link must wrap the models, not being before:
```xml
<a type="object" name="action_brand_model">
    <field name="model_count"/> MODELS
</a>
```